src/cljs/org/broadinstitute/firecloud_ui/page/workspaces_list.cljs

```diff
@@ -108,17 +116,24 @@
                    {:text "Running" :pred #(= "Running" (:status %))}
                    {:text "Exception" :pred #(= "Exception" (:status %))}]
          :columns
-         [{:header [:div {:style {:marginLeft -6}} "Status"] :starting-width 60
-           :content-renderer (fn [data] [StatusCell {:data data}])
-           :filter-by :none}
-          {:header "Workspace" :starting-width 450
-           :content-renderer (fn [data] [WorkspaceCell {:data data}])
-           :filter-by :name}
+         [{:sort-by (fn [m] (:status m))
+           :header [:div {:style {:marginLeft -6}} "Status"] :starting-width 60
+           :content-renderer (fn [data] [StatusCell {:data data}]) :filter-by :none}
+          {:sort-by (fn [m] (:name m))
+           :header "Workspace" :starting-width 450
+           :content-renderer (fn [data] [WorkspaceCell {:data data}]) :filter-by :name}
           {:header "Description" :starting-width 400
            :content-renderer (fn [description]
                                [:div {:style {:padding "1.1em 0 0 14px"
                                               :fontStyle (when-not description "oblique")}}
-                                (or description "No description provided")])}]
+                                (or description "No description provided")])}
+          {:header "Access Level" :starting-width 400 :sort-by WSAccessLevelKeyFunc
+           :sort-initial :asc
+           :content-renderer
+           (fn [accessLevel]
+             [:div {:style {:padding "1.1em 0 0 14px"
+                            :fontStyle (when-not accessLevel "oblique")}}
```

Comment:
Why is this font style here?  If there's no `accessLevel` there won't be any text to see.